config/manifests/default.yaml

```diff
@@ -253,17 +251,9 @@ metadata:
   name: meshery-operator-role
 rules:
 - apiGroups:
-  - meshery.layer5.io
-  - ""
-  - apps
+  - '*'
   resources:
-  - pods
-  - services
-  - configmaps
-  - deployments
-  - statefulsets
-  - brokers
-  - meshsyncs
+  - '*'
```

Comment:
In the future, this will be an area that users will want to be very specific on.

Reply:
Yes, this can be from OPA. When we request for specific set of resources, right now its all.